packages/io/json-deserializer/rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import banner from 'rollup-plugin-banner'
+import versionInjector from 'rollup-plugin-version-injector';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
+export default {
+  input: 'src/index.js',
+
+  output: [
+    {
+      file: 'dist/jscad-json-deserializer.min.js',
+      format: 'umd',
+      name: 'jscadJsonDeserializer',
+    },
```

Comment:
Most other io packages have a global `jscadModeling` export. Why the inconsistency? I would prefer to not use globals if not necessary.

Reply:
Hmm... only depends on array-utils.